packages/react/src/components/media/players/HlsPlayer.tsx

```diff
@@ -68,7 +70,11 @@ export const HlsPlayer = React.forwardRef<HTMLVideoElement, HlsPlayerProps>(
       const element = store.getState()._element;
       if (element && canUseHlsjs && !canPlayAppleMpeg && src.src) {
         const onError = (error: HlsError) => {
-          console.warn(error.response?.data.toString());
+          const cleanError = new Error(error.response?.data.toString());
+          if (isAccessControlError(cleanError)) {
+            onAccessControlError?.(cleanError);
```

Comment:
We will need to test this flow on Safari as well - my hunch is that we will need to hook into the native Safari error callback too

Reply:
Retesting I realise that the Offline error works also on Safari (using the video html element). But the Private error that was HlsPlayer specific. I will try to hook up error management to the video element too.

Reply:
@0xcadams You can check the [video player component](packages/react/src/components/media/Player.tsx) where I added error capture, but it does not work as expected.

First, there is error capture on the video element, which does not return enough information to understand what is the error about.
Second, there is error capture on the video's source element. This one is never triggered instead...

Can you look into this issue? Maybe based on your experience you know how to squeeze info out of the video/source elements...

Reply:
Yeah I can look into that, I also don't know how much info we can get from the native video player in Safari. We also have a lot of playback issues with Safari, so I'm actually leaning towards using HLS.js in Safari to make our Player easier to maintain. I think we could merge this without Safari and then follow up separately to move Safari playback to use HLS.js.

Reply:
https://github.com/livepeer/livepeer.js/issues/222

Reply:
I agree with you that using HLS.js always will simplify maintenance of the player. Great!